src/Common/tests/System/Net/Configuration.Http.cs

```diff
@@ -39,6 +39,8 @@ public static partial class Http
             public static string SelfSignedCertRemoteServer => GetValue("COREFX_HTTPHOST_SELFSIGNEDCERT", "https://self-signed.badssl.com/");
             public static string RevokedCertRemoteServer => GetValue("COREFX_HTTPHOST_REVOKEDCERT", "https://revoked.grc.com/");
 
+            public static string EchoClientCertificateRemoteServer => GetValue("COREFX_HTTPHOST_ECHOCLIENTCERT", "https://corefx-net-tls.azurewebsites.net/EchoClientCertificate.ashx");
```

Comment:
>Where is the code for this corefx-net-tls server?
Same private repo

>Also, why do we need yet another server name?
We need another name because it is implemented using Azure Web Apps. The current Azure endpoing (corefx-net.cloudapp.net) is using Azure Cloud Service (classic) Web Role).

Client certificates were difficult to implement in Web Role but easier in Azure Web App (formerly known as Azure websites).